src/plugins/intel_cpu/src/infer_request.cpp

```diff
@@ -27,6 +27,8 @@
 #include <ie_ngraph_utils.hpp>
 #include "proxy_mem_mgr.h"
 
+#include "../../inference/src/dev/make_tensor.hpp"
```

Comment:
I suppose we can move definition of tensor_to_blob to another file to avoid relative paths in includes.
E.g. to this place https://github.com/openvinotoolkit/openvino/blob/master/src/inference/dev_api/openvino/runtime/make_tensor.hpp

CC @ilyachur